openstudiocore/src/utilities/data/TimeSeries.cpp

```diff
@@ -626,11 +765,40 @@ namespace openstudio{
 
     std::shared_ptr<TimeSeries_Impl> TimeSeries_Impl::operator*(double d) const {
 
-      return std::shared_ptr<TimeSeries_Impl>(new TimeSeries_Impl(m_firstReportDateTime, 
-        m_secondsFromFirstReport,
-        m_values*d, 
+      return std::shared_ptr<TimeSeries_Impl>(new TimeSeries_Impl(m_startDateTime,
+        m_secondsFromStart,

```

Comment:
@jasondegraw I think this should still be the first way, constructed with m_firstReportDateTime, and m_secondsFromFirstReport, right?